src/webgpu/api/validation/render_pipeline/depth_stencil_state.spec.ts

```diff
@@ -32,6 +32,45 @@ g.test('format')
     t.doCreateRenderPipelineTest(isAsync, !!info.depth || !!info.stencil, descriptor);
   });
 
+g.test('depthCompare')
+  .desc(`The depthCompare in depthStencilState is optional for stencil-only formats.`)
+  .params(u => u.combine('isAsync', [false, true]).combine('format', kTextureFormats))
+  .beforeAllSubcases(t => {
+    const { format } = t.params;
+    const info = kTextureFormatInfo[format];
+    t.skipIfTextureFormatNotSupported(format);
+    t.selectDeviceOrSkipTestCase(info.feature);
+  })
+  .fn(t => {
+    const { isAsync, format } = t.params;
+    const info = kTextureFormatInfo[format];
+    const depthWriteEnabled = !!info.depth;
```

Comment:
This is weird, can't we just always set false for this value? This way if this is a stencil only format we don't have a validation error. What we really care about testing is depthCompare, and this member is not mentioned a single time in the test body. Maybe we could have a params for it that checks both `undefined` and a value.

Reply:
With the suggested changes, `const depthWriteEnabled = false;` works great!